packages/lib-user/src/components/shared/BarChart/BarChart.js

```diff
@@ -0,0 +1,98 @@
+import { DataChart, Text } from 'grommet'
+import { arrayOf, number, shape, string } from 'prop-types'
+import withResponsiveContext from '@zooniverse/react-components/helpers/withResponsiveContext'
+
+import getDateRangeLabel from './helpers/getDateRangeLabel'
+
+function BarChart ({
+  data = [],
+  dateRange = 'Last7Days',
```

Comment:
Suggestion to create a map similar to `asyncStates` where you define the string options once, and then access the  `dateRange` map from anywhere in lib-user. `dateRange` will likely be passed to BarChart from a parent that contains the range options dropdown, and all of the range options are already used in BarChart's story too. The prop initialization here would look something like `dateRange.Last7Days`. Another example of this is [subjectViewers.js](https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-classifier/src/helpers/subjectViewers/subjectViewers.js).

Reply:
Yes, great suggestion, will do.